iv-e/non-iv-e-medical.vbs

```diff
@@ -122,10 +127,11 @@ If action_option = "Close" then
 		LOOP UNTIL err_msg = ""
         CALL check_for_password(are_we_passworded_out)			'function that checks to ensure that the user has not passworded out of MAXIS, allows user to password back into MAXIS
     Loop until are_we_passworded_out = false					'loops until user passwords back in
+	Call check_for_MAXIS(False)
 
 	'The case note
     start_a_blank_case_note      'navigates to case/note and puts case/note into edit mode
-    Call write_variable_in_CASE_NOTE("**MA Needy child closed effective " & effective_date & "**")
+    Call write_variable_in_CASE_NOTE("**Non IV-E Medical child closed effective " & effective_date & "**")
     Call write_bullet_and_variable_in_CASE_NOTE("Reason(s)", reason_close)
     Call write_bullet_and_variable_in_CASE_NOTE("placement ended", placement_ended)
```

Comment:
Capitalize "placement ended" ???

Reply:
Yes, I capitalized the 'P' to align with the convention in the rest of this note.